public/static/app/perf-bb/SignalProcessingCtrl.js

```diff
@@ -24,23 +24,17 @@ mciModule.controller('SignalProcessingCtrl', function(
   // Holds currently selected items
   vm.selection = [];
 
-  // Could not be overridden by persistent user settings
-  const mandatoryDefaultFiltering = {
+  vm.defaultChangePointFilter = {
     create_time: '>' + moment().subtract(2, 'weeks').format(FORMAT.ISO_DATE),
     project: '=' + $window.project,
-  };
-
-  // Might be overridden by persistent user settings
-  const secondaryDefaultFiltering = {
```

Comment:
This secondary filter doesn't seem to do anything useful, so I just combined it with the mandatory default filtering.

Reply:
What happens if a user changes the value in the table?

![image](https://user-images.githubusercontent.com/22506/71919161-e28f4500-317b-11ea-9d04-408dd567cb19.png)

Both in terms of the default when no value is specified and keeping a use specified value?

If it behaves the same then less code is better.

Reply:
The behavior seems the same. Looking at the code, `mandatoryDefaultFiltering` and `secondaryDefaultFiltering` are only used in the `getDefaultFiltering` function. So, it doesn't seem like the comments about them are accurate. My guess is that someone was planning on adding that type of functionality in the future, but never got around to implementing it.

Reply:
kk, I'm prety sure that sure that the BB team do change the filtering on probability occasionally.